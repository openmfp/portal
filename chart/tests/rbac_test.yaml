suite: rbac
templates:
  - rbac.yaml
values:
  - ../test-values.yaml
tests:
  - it: match the snapshot
    asserts:
      - matchSnapshot: {}
