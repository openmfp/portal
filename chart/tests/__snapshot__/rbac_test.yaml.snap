match the snapshot:
  1: |
    apiVersion: rbac.authorization.k8s.io/v1
    kind: ClusterRole
    metadata:
      name: contentconfigurations-reader
    rules:
      - apiGroups:
          - core.openmfp.io
        resources:
          - contentconfigurations
        verbs:
          - get
          - watch
          - list
  2: |
    apiVersion: rbac.authorization.k8s.io/v1
    kind: ClusterRoleBinding
    metadata:
      name: read-contentconfigurations-global
    roleRef:
      apiGroup: rbac.authorization.k8s.io
      kind: ClusterRole
      name: contentconfigurations-reader
    subjects:
      - kind: ServiceAccount
        name: RELEASE-NAME
        namespace: NAMESPACE
